src/sentry/static/sentry/app/components/onboardingWizard/task.tsx

```diff
@@ -0,0 +1,287 @@
+  margin: 0;
+`;
+
+const ActionBar = styled('div')`
+  height: 40px;
+  border-top: 1px solid ${p => p.theme.borderLight};
+  margin: ${space(3)} -${space(3)} -${space(3)};
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 0 ${space(2)};
+`;
+
+const InProgressIndicator = styled(
+  ({user, ...p}: React.HTMLAttributes<HTMLDivElement> & {user?: AvatarUser | null}) => (
+    <div {...p}>
+      <Tooltip
+        disabled={!user}
+        containerDisplayMode="flex"
+        title={tct('This task has been started by [user]', {
+          user: user?.name,
+        })}
+      >
+        <IconEvent />
+      </Tooltip>
+      {t('Task in progress...')}
+    </div>
+  )
+)`
+  font-size: ${p => p.theme.fontSizeMedium};
+  font-weight: bold;
+  color: ${p => p.theme.yellowOrange};
+  display: grid;
+  grid-template-columns: max-content max-content;
+  align-items: center;
+  grid-gap: ${space(1)};
+`;
+
+const CTA = styled('div')`
+  color: ${p => p.theme.blue};
+  font-size: ${p => p.theme.fontSizeMedium};
+  font-weight: bold;
+`;
+
+const SkipButton = styled(Button)`
+  color: ${p => p.theme.gray2};
+`;
+
+const TaskLockedIndicator = styled(IconLock)``;
+
+TaskLockedIndicator.defaultProps = {
+  size: 'xs',
+  color: theme.redLight,
+};
+
+const PosedItemComplete = posed(Card)({
+  complete: {staggerChildren: 500},
+});
+
+const ItemComplete = styled(PosedItemComplete)`
+  cursor: pointer;
+  color: ${p => p.theme.gray3};
+  padding: ${space(1)} ${space(1.5)};
+  display: grid;
+  grid-template-columns: max-content 1fr max-content 20px;
+  grid-gap: ${space(1)};
+  align-items: center;
+`;
+
+const completedItemPoses = {
+  completeInit: {
+    opacity: 0,
+    x: -10,
+  },
+  complete: {
+    opacity: 1,
+    x: 0,
+  },
+};
+
+const CompleteIndicator = posed(IconCheckmark)(completedItemPoses);
+CompleteIndicator.defaultProps = {
+  circle: true,
+  color: theme.green,
+};
+
+const SkippedIndicator = posed(IconClose)(completedItemPoses);
+SkippedIndicator.defaultProps = {
+  circle: true,
+  color: theme.yellowOrange,
+};
+
+const CompletedDate = styled(posed.div(completedItemPoses))`
+  color: ${p => p.theme.gray2};
+  font-size: ${p => p.theme.fontSizeSmall};
+`;
+
+const TaskUserAvatar = posed(Avatar)(completedItemPoses);
+
+const TaskBlankAvatar = styled(posed(LetterAvatar)(completedItemPoses))`
+  position: unset;
+`;
+
+// XXX(epurkhiser): The withRouter HoC has incorrect typings. It does not
+// correctly remove the WithRouterProps from the return type of the HoC, thus
+// we manually have to do this.
+type PropsWithoutRouter = Omit<Props, keyof ReactRouter.WithRouterProps>;
+
+const WrappedTask = withOrganization(
+  ReactRouter.withRouter(Task) as React.ComponentClass<PropsWithoutRouter>
+);
+
+export default React.forwardRef<
+  HTMLDivElement,
+  Omit<React.ComponentProps<typeof WrappedTask>, 'forwardedRef'>
+>((props, ref) => <WrappedTask forwardedRef={ref} {...props} />);
```

Comment:
idk if it's just me, and this is a non-blocker, but I find this file difficult to read with all of the styled components.

Reply:
What would you suggest?